libs/atqr/domain/src/lib/challenge/challenge.entity.ts

```diff
@@ -4,9 +4,10 @@ import { dateDiff } from '../../utils/dateDifference';
 import { EmailAddress } from '../EmailService';
 import { PaymentMethodEntity } from '../PaymentMethod';
 import { Player } from '../player/player.entity';
+import { ChallengeStatus } from '../types/enums';
 
 export enum SupervisorEnum {
```

Comment:
this enum should be deleted since all enums are in enums.ts (by the way, this specifically is an old one not in use anymore, it was turned into ParticipationStatus enum).

Reply:
Yeah, this supervisor enum confused me for a bit and I didn't know what to use instead. I thought about ParticipationStatus but wasn't sure. I'll make the change and tag you again

Reply:
Actually, I'll just remove the enum and leave the code broken purposefully. Refactoring challenge is out of the scope of this issue. And I have a couple more issues to resolve before refactoring chalenge's props and constructor

Reply:
My reasoning is that challenge already is broken so let it be broken for a few more commit so I can have a sturdier base to refactor from.

Reply:
If you give me the okay I'll push it and we squash and merge

Reply:
Ignore the previous messages, I was messing up

Reply:
Done!